db/migrate/20250123220146_add_json_api_compatibility_to_client_configs.rb

```diff
@@ -0,0 +1,5 @@
+class AddJsonApiCompatibilityToClientConfigs < ActiveRecord::Migration[7.2]
+  def change
+    add_column :client_configs, :json_api_compatibility, :boolean, default: true
```

Comment:
We should add `null: false` here to avoid the possibility of a boolean with three states

Reply:
👍